src/ui/components/map/providers/mapboxmapprovider.js

```diff
@@ -11,6 +13,16 @@ import DOM from '../../../dom/dom';
  * @extends MapProvider
  */
 export default class MapBoxMapProvider extends MapProvider {
+  constructor (opts = {}, systemOpts = {}) {
+    super(opts, systemOpts);
+
+    /**
+     * Language of the map. Defaults to 'en'
+     * @type {string}
+     */
+    this._language = this._locale.substring(0, 2) || 'en';
```

Comment:
Is the `|| 'en'` default needed here? Will `_getValidatedLocale` take care of that default already? I think at this point in the code, `this._locale` will be defined (and will have at least two characters).

Reply:
Ya, that's a good point. It's not needed. I'll take it out.